path.bash.inc

```diff
@@ -1,5 +1,20 @@
 #!/bin/bash
 
 # Get the installation location of the fdk formula
+# Check for fdk
 installation_location=$(brew info fdk | grep Cellar | cut -d ' ' -f 1)
-export PATH="$installation_location/custom_node_location/bin:$PATH"
\ No newline at end of file
+if [[ -n "$installation_location" ]]; then
+    export PATH="$installation_location/custom_node_location/bin:$PATH"
+    echo "fdk found and added to PATH"
+else
+    echo "fdk not found"
```

Comment:
Please add better comments here since its customer facing it should be proper message